readthedocs/projects/admin.py

```diff
@@ -222,6 +229,56 @@ def build_default_version(self, request, queryset):
 
     build_default_version.short_description = 'Build default version'
 
+    def reindex_active_versions(self, request, queryset):
+        """Reindex all active versions of the selected projects to ES."""
+        qs_iterator = queryset.iterator()
+        for project in qs_iterator:
+            version_qs = Version.objects.filter(project=project)
+            active_versions = version_qs.filter(active=True)
+
+            if not active_versions.exists():
+                self.message_user(
+                    request,
+                    'No active versions of project {}'.format(project),
+                    messages.ERROR
+                )
+            else:
+                for version in active_versions.iterator():
+                    html_objs = HTMLFile.objects.filter(project=project, version=version)
+                    _indexing_helper(html_objs, wipe=False)
```

Comment:
This is reindexing a number of times, it should only run once.

Reply:
Thank you.
I have made and pushed the changes.